### DISCUSSION TITLE
Update issue

### DISCUSSION THREAD

**Discussion by bass225** [petrleocompel/hikaxpro_hacs#116] [General]
After HA update arming on the ax stopped working and only disarm still works. i get the attached error but does not matter how i change the settings the error persists: 

Failed to call service alarm_control_panel/alarm_arm_away. Validation error: Arming requires a code but none was given for alarm_control_panel.flat

also can't seem to update to latest V2.1.2 and its stuck on the 2.1.0-beta

Please Help

**Comment by petrleocompel** [comment]
Seems to be related to #118 - try again if it is fine close the discussion @bass225
